<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>LogIn</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="Layout_login">
         <item>
          <layout class="QHBoxLayout" name="layout_ip">
           <item>
            <widget class="QLabel" name="label_ip">
             <property name="text">
              <string>IP de la base de donnée</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_ip"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layout_username">
           <item>
            <widget class="QLabel" name="label_username">
             <property name="text">
              <string>Username de la base de donnée </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_username"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layout_password">
           <item>
            <widget class="QLabel" name="label_password">
             <property name="text">
              <string>Mot de passe de la base de donée</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_password">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layout_name">
           <item>
            <widget class="QLabel" name="label_name">
             <property name="text">
              <string>Nom de la base de donée</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_name"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="button_login">
           <property name="text">
            <string>LogIn</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
